packages/react/src/components/Modal/ConfirmationModal.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { Modal, ModalSize } from ":/components/Modal/index";
+import { useCunningham } from ":/components/Provider";
+import { Button } from ":/components/Button";
+import { DecisionModalProps } from ":/components/Modal/ModalProvider";
+
+export type ConfirmationModalProps = DecisionModalProps;
+
+export const ConfirmationModal = ({
+  title,
+  children,
+  onDecide,
+  ...props
+}: ConfirmationModalProps) => {
+  const { t } = useCunningham();
+  return (
+    <Modal
+      title={title ?? t("components.modals.helpers.confirmation.title")}
+      size={ModalSize.SMALL}
+      actions={
+        <>
+          <Button
+            color="secondary"
+            fullWidth={true}
+            onClick={() => onDecide(null)}
+          >
+            {t("components.modals.helpers.confirmation.cancel")}
+          </Button>
+          <Button fullWidth={true} onClick={() => onDecide("yes")}>
+            {t("components.modals.helpers.confirmation.yes")}
+          </Button>
+        </>
```

Comment:
IMO, instead of string, we should return boolean value through onDecide callback`

Reply:
That's what I did first but if in the future we want to implement modal having multiple decisions we will get stuck, for instance:

- Cancel = `null`
- Save & Quit = `save_quit`
- Quit = `quit`
- Modal closed = `undefined`

Reply:
Oh right, smart! 
In order to explicit that, what do you think to create an enum with current supported values?